client/jetpack-onboarding/controller.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import ReactDom from 'react-dom';
+
+/**
+ * Internal Dependencies
+ */
+import JetpackOnboardingMain from './main';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import { setSection } from 'state/ui/actions';
+
+const removeSidebar = context => {
+	ReactDom.unmountComponentAtNode( document.getElementById( 'secondary' ) );
+	context.store.dispatch( setSection( null, { hasSidebar: false } ) );
+};
```

Comment:
Looking forward to when we'll be able to ditch this kind of multi-tree rendering helpers, i.e. when STR lands.